applications/nublado/values-idfdev.yaml

```diff
@@ -2,9 +2,13 @@ controller:
   googleServiceAccount: "nublado-controller@science-platform-dev-7696.iam.gserviceaccount.com"
   slackAlerts: true
   image:
-    tag: "tickets-DM-38795-queue"
-    pullPolicy: "Always"
+    tag: "0.4.0"
```

Comment:
Don't hard-code the tag; it should come from `Chart.yaml` (which needs to be updated with the new version; I don't think you've done that).